### ISSUE TITLE
repair: ranges option is not supported for tablet repair

### ISSUE THREAD

**Issue by asias** [scylladb/scylladb#17416]
The management tool, e.g., scylla manager, needs the ranges option to select which ranges to repair on a node to schedule repair jobs. The option is not support yet in the initial "Basic tablet repair support" PR.

**Comment by mykaul** [comment]
But do we need this? Scylla Manager can certainly adapt to what Tablets need, not vice-versa.

**Comment by asias** [comment]
> But do we need this? Scylla Manager can certainly adapt to what Tablets need, not vice-versa.

Yes, of course. Without this, there is no way for Scylla Manger to adapt.
